content/departments/people-talent/talent/internship/index.md

```diff
@@ -111,8 +111,12 @@ Wage and Hour Guidelines
 
 ## 2022 Internship Cohort
 
+### Meet the members
+
 ![Internship Cohort](https://storage.googleapis.com/sourcegraph-assets/internship%20cohort.png)
```

Comment:
FYI this is not accessible, because screen readers cannot read the text in the image. There should be individual images for each picture, with the text being actual text (not part of the image itself).